manifests/server/etcd/setup.pp

```diff
@@ -56,50 +97,61 @@
         ensure => 'absent',
       }
     }
-  } else {
-    package { 'etcd':
+
+    group { $group:
       ensure => $ensure,
-      name   => $package,
+      system => true,
+      gid    => $gid,
     }
-  }
 
-  user { 'etcd':
-    ensure => $ensure,
-  }
-  group { 'etcd':
-    ensure  => $ensure,
-    members => ['etcd'],
+    user { $user:
+      ensure     => $ensure,
+      comment    => 'ETCD user',
```

Comment:
A real nitpick, but 'etcd' should be lowercase

Reply:
will change that :)